perfkitbenchmarker/providers/gcp/gce_network.py

```diff
@@ -234,28 +243,136 @@ class GceNetwork(network.BaseNetwork):
   def __init__(self, network_spec):
     super(GceNetwork, self).__init__(network_spec)
     self.project = network_spec.project
-    name = FLAGS.gce_network_name or 'pkb-network-%s' % FLAGS.run_uri
-    mode = 'auto' if FLAGS.gce_subnet_region is None else 'custom'
+
+    #  Figuring out the type of network here.
+    #  Precedence: User Managed > CIDR (multi) > gce_subnet flag (single) > Auto (default)
+    self.net_type = 'default'  # Default network 'auto' mode. Single VPC with subnets in all regions
```

Comment:
Recommend using enum.

Reply:
Fixed. Added NetType(Enum) to network.py